tests/radar_client.unit.test.js

```diff
@@ -126,7 +128,7 @@ exports.RadarClient = {
 
       client._write = function(hash, fn) {
         called = true;
-        assert.deepEqual(hash, {
+        assert.deepEqual(hash.message, {
```

Comment:
OK.  This was in a few places, and I used request.getMessage() instead of hash.message.